<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sys_Export_Dia</class>
 <widget class="QDialog" name="Sys_Export_Dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to file</string>
  </property>
  <widget class="QGroupBox" name="label_header">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>393</width>
     <height>221</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Choose the type of export and the file for saving the data</string>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QRadioButton" name="radioButton_excel_tab">
      <property name="text">
       <string>output tab separated (e.g. Excel)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_excel_comma">
      <property name="text">
       <string>output comma separated (.csv, e.g. Excel/ParaView)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_tecplot">
      <property name="text">
       <string>output tecplot format</string>
      </property>
     </widget>
    </widget>
    <widget class="Sys_One_Filechooser_Wid" name="file_chooser" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="selectAll_GroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Which Rows should be exported?</string>
   </property>
   <widget class="QCheckBox" name="selectAll_checkBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>25</y>
      <width>97</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Select all rows?</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>300</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>208</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sys_One_Filechooser_Wid</class>
   <extends>QWidget</extends>
   <header location="global">sys_one_filechooser_wid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
